packages/server/modules/gatekeeper/domain/billing.ts

```diff
@@ -173,7 +173,12 @@ export type ReconcileSubscriptionData = (args: {
   applyProrotation: boolean
 }) => Promise<void>
 
-export type WorkspaceSeatType = 'viewer' | 'editor'
+export const WorkspaceSeatType = <const>{
+  Viewer: 'viewer',
+  Editor: 'editor'
+}
+export type WorkspaceSeatType =
+  (typeof WorkspaceSeatType)[keyof typeof WorkspaceSeatType]
```

Comment:
I would like to avoid these kind of changes.
I thought we agreed that union string types are fine

cc
@gjedlicska

Reply:
it's the same type it was before, except now you can also use it as a runtime enum. I don't see what the problem is with that

Reply:
i think magic string literals are bad practice - 1) the autocomplete only works when you're putting them into a constant/param that is already typed as the expected type. if you're just doing `const a = ...` and start typing the string, its not gonna know `a`'s expected type and you won't get autocomplete. don't have that issue with enums 
2) duplicating strings over and over again makes it more difficult to change the value in the future

that said, i'm ok with someone here disagreeing - that's why there's both options available. you can still do the magic strings if you want, but at least there's the option to use the enum if you prefer that